docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -156,8 +156,6 @@ data:
   server.connection.status.cache.expiration: "1h0m0s"
   # Cache expiration for OIDC state (default 3m0s)
   server.oidc.cache.expiration: "3m0s"
-  # Cache expiration for failed login attempts (default 24h0m0s)
-  server.login.attempts.expiration: "24h0m0s"
```

Comment:
Setting this was actually a no-op. Need to investigate more. Maybe we switched from an in-memory cache to a Redis cache and just didn't update the code?

Reply:
This PR replaced the redis-based failures counter with an in-memory solution that doesn't have a configurable expiration: https://github.com/argoproj/argo-cd/pull/5477

Not going to reevaluate that decision for this PR, just going to remove the dead code.